Postgres/create_tables.sql

```diff
@@ -1,33 +1,48 @@
 CREATE EXTENSION hstore;
 SET timezone = 'Australia/Sydney';
 
+CREATE TABLE groups (
+  UID   SERIAL PRIMARY KEY,
+  Name  VARCHAR(50) NOT NULL,
+  Permission int UNIQUE NOT NULL
```

Comment:
May be worth adding a procedure or something which handles the permission level - this will be pretty annoying to manage if it persists to the end product.

Reply:
yep sure

Reply:
also, enums will make it easier to maintain: https://www.postgresql.org/docs/9.1/datatype-enum.html

Reply:
so 
```plpgsql
CREATE TYPE permissions_enum AS ENUM ('read', 'write', 'delete');
# ... 
# ...
Permission permissions_enum UNIQUE NOT NULL
```
or something